ampd/ampd-release.md

```diff
@@ -0,0 +1,66 @@
+# AMPD Release Process
+
+**For maintainers only**
+
+This document outlines the step-by-step process for releasing a new version of AMPD. This guide is intended for team members who are responsible for managing releases.
+
+## Prerequisites
+
+- Access to the [axelar-amplifier](https://github.com/axelarnetwork/axelar-amplifier) repository
+- Access to the private [infrastructure](https://github.com/axelarnetwork/infrastructure) repository
+
+## Release Process
+
+### 1. Update and Tag Release Version
+
+1. Navigate to the [Update and tag release version](https://github.com/axelarnetwork/axelar-amplifier/actions/workflows/release.yaml) GitHub Action.
+2. Run the workflow with the following:
+   - Use workflow from: `main` branch
+   - Enter `ampd` as the package input
+   - **IMPORTANT**: For the first run, enable "Dry run" to verify everything works correctly
+
+3. After confirming the dry run completed successfully, run the workflow again with "Dry run" disabled to create the actual release tag.
+
+### 2. Update the Changelog
+
+1. Create a PR to update the root level `CHANGELOG.md` file based on the newly created tag from step 1.
+2. Update the changelog header section with the new version information.
+
+Example changes:
+
+```diff
+- [Full Changelog](https://github.com/axelarnetwork/axelar-amplifier/compare/ampd-v1.5.0..HEAD)
++ [Full Changelog](https://github.com/axelarnetwork/axelar-amplifier/compare/ampd-v1.5.1..HEAD)
++ 
++ ## [v1.5.1](https://github.com/axelarnetwork/axelar-amplifier/tree/ampd-v1.5.1) (2025-03-26)
++ 
++ [Full Changelog](https://github.com/axelarnetwork/axelar-amplifier/compare/ampd-v1.5.0..ampd-v1.5.1)
++ 
++ - Fix arithmetic operations with XRPLTokenAmount [#780](https://github.com/axelarnetwork/axelar-amplifier/pull/780)
+```
+
+3. Create a PR with these changes and merge it to the `main` branch.
+
+### 3. Build the AMPD Release
+
+1. Navigate to the [Build and release binary and image](https://github.com/axelarnetwork/axelar-amplifier/actions/workflows/build-ampd-release.yaml) GitHub Action.
+2. Run workflow using the newly created tag as input.
+3. Verify that a new release appears under the Releases section of the amplifier repository.
+
+### 4. Deploy to Network
+
+#### 4.1 Update Infrastructure Configuration
+
+1. Create a PR in the private [infrastructure](https://github.com/axelarnetwork/infrastructure) repository to update the AMPD tag in the Helm charts for your target network (devnet, testnet, or stagenet).
+
+   Example PR for reference: [link](https://github.com/axelarnetwork/infrastructure/commit/e7dc80160404b75ac8e3b850d834a53e76680eab)
+
```

Comment:
This is an internal repo. There should be an FAQ on how to deploy internally in notion, instead of the public repo

Reply:
I talked to @maancham about that. I still prefer keeping it on the repo, because in notion it will just get lost and not get updated. I don't see any security issues with this, can you elaborate on your concerns?

Reply:
Generally a bad security practice to discuss internal infra details in public repos. We should create a FAQ notion page for the team that's more easily discoverable. I'd also be ok with linking to that notion page here, but at least the internal details are encapsulated